index.html

```diff
@@ -933,23 +967,24 @@ <h2>Generic DID URL Parameters</h2>
         </table>
 
         <p>
-  The exact processing rules for these parameters are specified in
-  [[?DID-RESOLUTION]].
+Additional considerations for processing these parameters are provided
+in [?DID-RESOLUTION].
```

Comment:
Suggest this change to align with other mentions. This line wasn't part of the original pull request, but we should change it now to keep it consistent. 

```suggestion
Additional considerations for processing these parameters are discussed
in [?DID-RESOLUTION].
```